test/e2e/cassandracluster_test.go

```diff
@@ -2,13 +2,19 @@ package e2e
 
 import (
 	goctx "context"
+	"fmt"
+	v1 "k8s.io/api/apps/v1"
+	"os"
 	"testing"
+	"time"
 
-	framework "github.com/operator-framework/operator-sdk/pkg/test"
-	"github.com/stretchr/testify/assert"
 	"github.com/Orange-OpenSource/cassandra-k8s-operator/pkg/apis"
 	api "github.com/Orange-OpenSource/cassandra-k8s-operator/pkg/apis/db/v1alpha1"
 	mye2eutil "github.com/Orange-OpenSource/cassandra-k8s-operator/test/e2eutil"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	log "github.com/sirupsen/logrus"
```

Comment:
We use `logrus` everywhere else, we should keel doing it here to be consistent.

Reply:
@jsanda did you miss my comment ☝️ ?
I propose to keep using `logrus` any not `log` here as we do in other files

Reply:
I missed or forgot. You reminder is very timely though :) 

I am in the process of updating the test to be more consistent with other e2e tests. I am changing the test to create the cluster from the yaml file as is done in other tests. 

I will update the logging as well and hopefully commit/push shortly.

Reply:
Good to hear ! Thanks for you help again ;)